tests/rpmbuild.at

```diff
@@ -866,6 +866,7 @@ RPMTEST_CHECK([
 RPMDB_INIT
 
 runroot rpmbuild -bb --quiet \
+		--define '__local_file_attrs local_generator' \
```

Comment:
While this is "just test case", maybe the `local_generator` could use different name, just to avoid the impression that the `local` is somehow mandatory in the latter

Reply:
Maybe something like `totally_random_generator_name`